app/middlewares/logger/errorLogger.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-unused-vars */
+import { logger } from './logger';
+import { getLogText } from './utils';
+
+export const errorLogger = (err, req, res, next) => {
+  const logText = getLogText(req);
+
+  logger.error(`Error: ${err.message}. Happened with this request: ${logText}`);

```

Comment:
Read task 5.2  - middleware should log all unhandled errors (but you only logging error messages).

Reply:
Logging error without a call stack is useless

Reply:
Fixed it, thanks